install-apps-via-chocolatey.ps1

```diff
@@ -0,0 +1,27 @@
+# Note: upgrade is the recommended choco subcommand / verb for scripted installations
+
+$ToInstall = @(
+    '7zip',
+    'adoptopenjdk',
+    'boostnote',
+    'filezilla',
+    'Firefox',
+    'git',
+    'GoogleChrome',
+    # 'libreoffice-fresh',
+    'microsoft-teams',
+    'notepadplusplus',
```

Comment:
I would say that boostnote, microsoft-teams, notepad++ and even vscode (even though I use it, I know a lot of people don't like it) are really subjective.

Reply:
Regarding Teams, I know FUSD uses it for a lot of their communication, so that could be important. Maybe a note to grab teams if you're on anything FUSD related?